packages/flutter/test/rendering/proxy_box_test.dart

```diff
@@ -732,10 +732,10 @@ void main() {
 
   test('AnimatedOpacity sets paint matrix to zero when alpha == 0 (sliver)', () {
     final RenderSliver sliver = RenderSliverToBoxAdapter(child: RenderConstrainedBox(additionalConstraints: const BoxConstraints.tightFor(width: 20)));
-    final RenderBox parent = RenderConstrainedBox(additionalConstraints: const BoxConstraints.tightFor(width: 20));
+    final RenderSliverPadding parent = RenderSliverPadding(padding: const EdgeInsets.all(4));
```

Comment:
This change is fine, the test only needed to be sure to _have_ a child, doesn't really matter what the type is.